tests/publish/nitf_formatter_tests.py

```diff
@@ -0,0 +1,42 @@
+# -*- coding: utf-8; -*-
+#
+# This file is part of Superdesk.
+#
+# Copyright 2013, 2014 Sourcefabric z.u. and contributors.
+#
+# For the full copyright and license information, please see the
+# AUTHORS and LICENSE files distributed with this source code, or
+# at https://www.sourcefabric.org/superdesk/license
+
+from superdesk.tests import TestCase
+from superdesk.publish.formatters.nitf_formatter import NITFFormatter
+from superdesk.publish.formatters import Formatter
+from superdesk.publish import init_app
+
+
+class NitfFormatterTest(TestCase):
```

Comment:
This should extend from `SuperdeskTestCase` instead of `TestCase`. `SuperdeskTestCase`  takes care of `init_app()`